changelog.md

```diff
@@ -1,3 +1,5 @@
 # Changelog
 
 ## Unreleased
+
+- respect `toc-depth` in the HTML format (bootstrap) rather than always acting as if depth is 3.
```

Comment:
Changelog should be a separate commit (in case we want to cherry pick back onto v1.0)